HW1/task1.py

```diff
@@ -0,0 +1,10 @@
+#Задача 2: Найдите сумму цифр трехзначного числа.
+#*Пример:*
+#123 -> 6 (1 + 2 + 3)
+#100 -> 1 (1 + 0 + 0) |
+
+n= int(input('Введите число:'))
+a= int(n/100)
+b= int((n/10)%10)
+c= int(n%10)
```

Comment:
Во первых, не забывайте приводить код к виду PEP8. Во-вторых, функцию `int` использовать не обязательно, достаточно использовать целочисленное деление. В третьих, операции деления имеют одинаковый приоритет, то есть скобки не нужны.
```suggestion
n = int(input('Введите число:'))
a = n // 100
b = n / 10 %10
c = n % 10
```